test/test_Regression/test_gpmsa_cobra.C

```diff
@@ -252,22 +252,21 @@ int main(int argc, char ** argv) {
   // Initial condition of the chain
   // Have to set each of these by hand, *and* the sampler is sensitive to these
   // values
-  paramInitials[0] = 0.5; // param 1
-  paramInitials[1] = 0.5; // param 2
-  paramInitials[2] = 0.5; // param 3
-  paramInitials[3] = 0.5; // param 4
-  paramInitials[4] = 0.5; // param 5
-  paramInitials[5]  = 0.4;  // not used.  emulator mean
```

Comment:
When we have enough outputs that we don't want to use every SVD term, then that's right.  I'm going to have to add some more helper/indexing APIs here, because with the "truncate based on remainder" use case, whether we have a truncation error to calibrate or not will actually be data dependent.